code/game/objects/items/devices/organ_module/module_hud.dm

```diff
@@ -0,0 +1,56 @@
+/obj/item/organ_module/active/hud
+	name = "Embedded Hud"
+	desc = "If you see this hud in game, congratulation!"
+	verb_name = "Ativate Hud"
+	icon_state = "hud"
+	matter = list(MATERIAL_STEEL = 3, MATERIAL_GLASS = 2)
+	allowed_organs = list(BP_HEAD)
+	var/obj/item/clothing/glasses/holding = null
```

Comment:
var/obj/item/clothing/glasses/holding

Reply:
What he means is "= null" isn't needed, new variables are always null if not initiated at the creation time